client/layout/store-alerts/index.js

```diff
@@ -104,18 +65,23 @@ class StoreAlerts extends Component {
 		const numberOfAlerts = alerts.length;
 		const alert = alerts[ currentIndex ] ? alerts[ currentIndex ] : null;
 		const type = alert && alert.type ? alert.type : null;
-		const icon = type && alertTypes[ type ] ? alertTypes[ type ].icon : null;
 		const className = classnames( 'woocommerce-store-alerts', {
-			'is-alert-emergency': 'emergency' === type,
-			'is-alert-alert': 'alert' === type,
-			'is-alert-critical': 'critical' === type,
+			'is-alert-error': 'error' === type,
+			'is-alert-update': 'update' === type,
 		} );
+		const actions =
```

Comment:
Should we do a `return` early, perhaps after `const { alerts } = this.props;` if `! alerts || alerts.length === 0`?